client/components/comment.vue

```diff
@@ -58,7 +58,7 @@
                   flat
                   icon
                   small
-                  @click.stop="onDeleteClicked"
+                  @click="onDeleteClicked"
```

Comment:
Why this change? I like removal of `stop`, because stopping propagation is generally problematic (because often one depends on propagation somewhere else, like to close modals). But why is this change necessary here?